internal/util/framing.go

```diff
@@ -19,7 +18,7 @@ func SafeReadLine(r *bufio.Reader) (line []byte, err error) {
 	return
 }
 
-// Exhaust all the identifiers in r,
+// Exhaust consumes all the identifiers in r,
 // The format of an indentifier is string\n
```

Comment:
what does this mean: `the format of an identifier is string\n`, do you want to say that each line contains an identifier, and it should end with `\n`?

Reply:
This is the intent.